readme.md

```diff
@@ -98,6 +98,50 @@ wp vuln theme-status
  * `--format=<format>` Accepted values: table, csv, json, count, ids, yaml. Default: table
  * `--nagios` Output for nagios
 
+### Example output
+
+#### Checking plugins and themes for reported vulnerabilities:
+
+```shell
+$ wp vuln status
+
+Vulnerability API Provider: WPScan
+WordPress 6.2.2
++-----------+-------------------+-----------------------------------------------------------+---------------+----------+
+| name      | installed version | status                                                    | introduced in | fixed in |
++-----------+-------------------+-----------------------------------------------------------+---------------+----------+
+| WordPress | 6.2.2             | No vulnerabilities reported for this version of WordPress | n/a           | n/a      |
++-----------+-------------------+-----------------------------------------------------------+---------------+----------+
+```
+
+Using the JSON format:
+
+```shell
+$ wp vuln status --format=json
+
+{"core":[{"name":"WordPress","installed version":"6.2.2","status":"No vulnerabilities reported for this version of WordPress","introduced in":"n\/a","fixed in":"n\/a"}],"plugins":
```

Comment:
Similar to above, should contain full json containing core, plugins and themes results.

```suggestion
{"core":[{"name":"WordPress","installed version":"6.2.2","status":"No vulnerabilities reported for this version of WordPress","introduced in":"n\/a","fixed in":"n\/a"}],"plugins":[{"name":"simple-podcasting","installed version":"1.3.0","status":"No vulnerabilities reported for this version of simple-podcasting","introduced in":"n\/a","fixed in":"n\/a"},{"name":"woocommerce","installed version":"7.8.2","status":"No vulnerabilities reported for this version of woocommerce","introduced in":"n\/a","fixed in":"n\/a"},{"name":"wordpress-seo","installed version":"20.2","status":"Wordpress Yoast SEO plugin <= 20.2 - Authenticated (Contributor+) DOM-Based Cross-Site Scripting vulnerability","introduced in":"<= 20.2","fixed in":"20.2.1"}],"themes":[{"name":"twentytwentythree","installed version":"1.1","status":"No vulnerabilities reported for this version of twentytwentythree","introduced in":"n\/a","fixed in":"n\/a"}]}
```